azurerm/internal/clients/client.go

```diff
@@ -118,6 +119,7 @@ type Client struct {
 	Portal           *portal.Client
 	Postgres         *postgres.Client
 	PrivateDns       *privatedns.Client
+	PowerBIDedicated *powerBIDedicated.Client
```

Comment:
What is this API for? looking in the portali just see "Power BI Embedded" and no mention of dedicated? Maybe this just wants to be
```suggestion
	PowerBI *powerBIDedicated.Client
```

Reply:
This PowerBI Capacity API is for provisioning capacity which is the container to load the power bi report and dataset. Maybe service owner wants to distinguish between powerbi capacity and powerbi workspace. But they are associated with PowerBI. So I think your answer is reasonable. we should align with the portal and docs as that is what the user is likey familiar with rather then the API & SDK. So I will updated name from PowerBIDedicated to PowerBI. Does it make sense? 
Below is my summary:
azure docs:
	power bi embedded  = powerbi capacity (https://docs.microsoft.com/en-us/power-bi/developer/azure-pbie-create-capacity)
	power bi workspace = powerbi workspace (https://docs.microsoft.com/en-us/power-bi/service-create-the-new-workspaces)
	
azure api spec:
	PowerBIDedicated = powerbi capacity (https://github.com/Azure/azure-rest-api-specs/blob/master/specification/powerbidedicated/resource-manager/Microsoft.PowerBIdedicated/stable/2017-10-01/powerbidedicated.json)
	powerbiembedded  = powerbi workspace (https://github.com/Azure/azure-rest-api-specs/blob/master/specification/powerbiembedded/resource-manager/Microsoft.PowerBI/stable/2016-01-29/powerbiembedded.json)